### PR TITLE
app-misc/kclock: new package

### PR DESCRIPTION
**Pull request by CyberTailor**

### CODE REVIEW

**Review by a17r — APPROVED** [review]
Minor nits as other two PRs.

Thanks for your work, looks solid. I'll try to get to build, install and check deps soon.

File: app-misc/kclock/kclock-23.04.49.9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+LICENSE="CC0-1.0 CC-BY-4.0 GPL-2+ GPL-3+ LGPL-2.1+"
+SLOT="0"
+KEYWORDS=""
+
+DEPEND="
+	>=dev-libs/kirigami-addons-0.6:5
+	>=dev-qt/qtdbus-${QTMIN}:5
+	>=dev-qt/qtdeclarative-${QTMIN}:5
+	>=dev-qt/qtgui-${QTMIN}:5
+	>=dev-qt/qtmultimedia-${QTMIN}:5
```
Comment (a17r) [inline app-misc/kclock/kclock-23.04.49.9999.ebuild:22]: missing `qml` USEdep here

File: app-misc/kclock/kclock-23.04.49.9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	>=dev-qt/qtwidgets-${QTMIN}:5
+	>=kde-frameworks/kconfig-${KFMIN}:5
+	>=kde-frameworks/kcoreaddons-${KFMIN}:5
+	>=kde-frameworks/kdbusaddons-${KFMIN}:5
+	>=kde-frameworks/ki18n-${KFMIN}:5
+	>=kde-frameworks/kirigami-${KFMIN}:5
+	>=kde-frameworks/knotifications-${KFMIN}:5
+	>=kde-frameworks/plasma-${KFMIN}:5
+"
+RDEPEND="${DEPEND}"
```
Comment (a17r) [inline app-misc/kclock/kclock-23.04.49.9999.ebuild:33]: missing `RDEPEND=">=dev-qt/qtgraphicaleffects-${QTMIN}:5"`

**Review by a17r — CHANGES_REQUESTED** [review]

File: app-misc/kclock/kclock-23.04.49.9999.ebuild (hunk 2 above)
Comment (CyberTailor) [reply app-misc/kclock/kclock-23.04.49.9999.ebuild]: Fixed

File: app-misc/kclock/kclock-23.04.49.9999.ebuild (hunk 1 above)
Comment (CyberTailor) [reply app-misc/kclock/kclock-23.04.49.9999.ebuild]: Fixed

**Review by a17r — APPROVED** [review]

**Comment by a17r** [comment]
https://bugs.gentoo.org/907251

**Pull request closed without merging**